PocketKit/Sources/PocketKit/MyList/EmptyStates/Search/ErrorEmptyState.swift

```diff
@@ -0,0 +1,17 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Foundation
+import Textile
+
+struct ErrorEmptyState: EmptyStateViewModel {
+    let imageAsset: ImageAsset = .warning
+    let maxWidth: CGFloat = Width.normal.rawValue
+    let icon: ImageAsset? = nil
+    let headline: String? = L10n.oopsTryAgain
+    let detailText: String? = L10n.WeReExperiencingAnErrorAndTemporarilyCanTGetYouWhatYouNeed.pleaseTryAgainLater
```

Comment:
our localization variable names are a bit long. However, I decided to use the current implementation as opposed to refactoring it as part of this work. We should discuss on what our stance is on this. Do we continue with this for now?

Reply:
Seems like we'll look at it as a followup - https://pocket.slack.com/archives/C01UH57EJKD/p1676391105038829?thread_ts=1676333516.460409&cid=C01UH57EJKD

Reply:
This is what @Gio2018 did https://github.com/Pocket/pocket-ios/pull/402/files#diff-adf47436386e17ed486299bf385bee0272f10cf98b3a79f17cbff3995cecdff4R141

Reply:
that's awesome! okay updated the keys, will merge after successful build